docs/src/main/sphinx/installation/cli.rst

```diff
@@ -48,15 +48,59 @@ recognizes these certificates.
 
 Authentication
 --------------
+Password/LDAP Authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can override your username with the ``--user`` option. It defaults to your
 operating system username. If your Trino server requires password
 authentication, use the ``--password`` option to have the CLI prompt for a
 password. You can set the ``TRINO_PASSWORD`` environment variable with the
 password value to avoid the prompt.
 
+When Trino is configured with LDAP authentication, invoking CLI requires additional options
+such as ``--keystore-*`` or ``--truststore-*`` to secure TLS connection.
+The simplest way to invoke the CLI is with a wrapper script.
```

Comment:
@mosabua I have tried to simplify these lines from the original documentation. I thought the original sentence was a little complicated to quickly understand. Let me know your thoughts on this.

Reply:
All this is actually not needed. The only thing really needed for password / ldap usage is --server and  --user. And that is only needed if different form the operating system user (which is often the case). 

Password will be prompted

catalog and schema are generally available, optional and should be documented in a separate section (in a separate PR)

all the keystore and truststore is only needed custom cert is used. That should also be covered in a separate section (in the TLS section higher above)